custom-platform/pack.php

```diff
@@ -5,7 +5,7 @@
  /****** REPLACE THESE  *******/
  $packageID = "uncon2017-custom-platform";
  $packageLabel = "UnCon 2017 Custom Platform Registration";
- $supportedVersionRegex = '^7.9.[\d]+.[\d]+$';
```

Comment:
We need to use something like the the following regex. It should support 7.9, 7.10, 7.11 and 8.0. With 8.0, we plan on only using Maj.Minor.Patch versioning. So we shouldn't always expect a 4th section in our version numbers.

`  ^(7\.(9|10|11)\.[\d]+.*|8\.[\d]+\.[\d]+.*)  `